include/rdma/fabric.h

```diff
@@ -570,6 +570,8 @@ enum {
 	FI_SETOPSFLAG,		/* uint64_t flags */
 	FI_ALIAS,		/* struct fi_alias * */
 	FI_GETWAIT,		/* void * wait object */
+	FI_TX_GETWAIT,		/* void * wait tx object */
+	FI_RX_GETWAIT,		/* void * wait rx object */
```

Comment:
Please remove these changes from the headers and have them provider specific, since they will be unneeded.  Also note that you can't insert new values into the middle of the enum.  That changes the values of the other entries, which breaks the ABI.